### PR TITLE
feature-request/ebs-default-encryption: Added 'EBS Default Encryption Enabled' plugin and test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

**Review by dipsubha06 — APPROVED** [review]

File: helpers/aws/enum.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    1: 'sse',
+    2: 'awskms',
+    3: 'awscmk',
+    4: 'externalcmk',
+    5: 'cloudhsm',
+};
+
+module.exports = {
+    encryptionLevelMap,
+};
```
Comment (giorod3) [inline helpers/aws/enum.js:18]: this logic can be added top the plugin since it is only being used once.

File: helpers/aws/functions.js [hunk 2]
```diff
@@ -272,6 +272,22 @@ let divideArray = function(array, size) {
...
+        }
+    }
+    if (kmsKey.Origin === 'EXTERNAL') {
+        return 4;
+    }
+    if (kmsKey.Origin === 'AWS_CLOUDHSM') {
+        return 5;
+    }
+}
+
```
Comment (giorod3) [inline helpers/aws/functions.js:290]: same here

File: plugins/aws/ec2/ebsDefaultEncryptionEnabled.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'EBS Encryption Enabled By Default',
+    category: 'EC2',
+    description: 'Ensure the setting for Encryption by default is enabled',
```
Comment (giorod3) [inline plugins/aws/ec2/ebsDefaultEncryptionEnabled.js:7]:
```suggestion
    description: 'Ensure the setting for encryption by default is enabled',
```

**Review by giorod3 — CHANGES_REQUESTED** [review]
lets talk about this in the morning

File: helpers/aws/enum.js (hunk 1 above)
Comment (giorod3) [reply helpers/aws/enum.js:18]: I see where you added this for other plugins. lets discuss

**Review by giorod3 — CHANGES_REQUESTED** [review]
change the map to array and add to index helper. example: encryptionLevel: []

**Review by giorod3 — APPROVED** [review]

**Pull request merged**